pkg/network/port_test.go

```diff
@@ -150,32 +152,19 @@ func TestReadInitialUDPState(t *testing.T) {
 	nsIno, err := kernel.GetInoForNs(ns)
 	require.NoError(t, err)
 
-	require.Eventually(t, func() bool {
+	require.EventuallyWithT(t, func(c *assert.CollectT) {
 		initialPorts, err := ReadInitialState("/proc", UDP, true)
-		require.NoError(t, err)
+		require.NoError(c, err)
 		for _, p := range ports[:2] {
-			if _, ok := initialPorts[PortMapping{testRootNs, p}]; !ok {
-				t.Errorf("PortMapping(testRootNs, p) returned false for port %d", p)
-				return false
-			}
+			assert.Contains(c, initialPorts, PortMapping{testRootNs, p}, fmt.Sprintf("PortMapping (testRootNs, p) returned false for port %d", p))
 		}
 		for _, p := range ports[2:] {
-			if _, ok := initialPorts[PortMapping{nsIno, p}]; !ok {
-				t.Errorf("PortMapping(nsIno, p) returned false for port %d", p)
-				return false
-			}
-		}
-
-		if _, ok := initialPorts[PortMapping{testRootNs, 999}]; ok {
-			t.Errorf("expected IsListening(testRootNs, 999) to return false, but returned true")
-			return false
+			assert.Contains(c, initialPorts, PortMapping{nsIno, p}, fmt.Sprintf("PortMapping(nsIno, p) returned false for port %d", p))
 		}
-		if _, ok := initialPorts[PortMapping{nsIno, 999}]; ok {
-			t.Errorf("expected IsListening(testRootNs, 999) to return false, but returned true")
-			return false
+		if isUnusedUDPPort(t, 53) {
```

Comment:
Port `53` is very likely to be in use as there is a separate change in the works to have a local dns server running.

Reply:
🤦 I was using 53 for testing. Thanks for catching. I'll switch back to 999